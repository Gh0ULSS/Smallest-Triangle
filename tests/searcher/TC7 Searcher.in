-4.76, 5.98
42.42, -3.43
0.78, 8.74
-31.53, 1.19
12.99, 29.3
-40.59, -19.66
-40.93, 30.96
19.34, -45.81
48.22, 46.48
15.39, 11.56
-34.25, -48.5
2.84, -44.04
-30.98, -25.81
-46.99, -3.61
-5.95, 34.24
1.91, 14.03
-0.02, 16.24
-4.27, -22.18
49.77, 49.57
34.02, 20.78
-18.47, -27.03
-21.1, -42.98
26.63, -9.96
34.66, -11.35
45.8, 34.73
-49.95, -29.03
48.04, -10.26
-42.7, 12.95
27.85, -23.02
-41.29, -16.74
46.41, 25.8
-38.2, -25.36
-39.9, -44.01
29.7, -32.23
5.93, -5.26
-30.93, 23.19
-36.9, 14.37
-38.35, -7.92
-28.71, -23.02
47.09, 30.34
-19.59, 38.49
-28.93, -10.57
35.44, 14.18
-39.97, 48.93
27.27, -17.1
-40.99, 8.27
-25.7, 10.13
-12.83, -4.68
45.91, -1.63
7.46, 36.65
-31.72, -34.59
40.84, 31.78
-25.05, -31.02
23.94, 44.04
-30.34, 45.01
38.22, 10.35
-7.85, -39.62
-46.13, 46.27
-26.16, 20.46
-24.3, 32.37
9.65, -20.66
-43.12, -27.16
5.94, 35.24
11.43, -21.98
41.74, -29.6
-48.34, -23.08
-5.43, -43.95
-32.37, -13.12
7.22, -36.84
-13.79, 39.09
48.05, 15.69
19.12, 8.44
-35.97, -46.49
-48.21, 41.02
20.1, 46.28
-47.87, 13.62
-1.78, 23.05
-18.11, 49.94
-42.47, 4.61
23.7, 40.02
23.71, 20.37
29.33, 41.5
-14.82, 18.51
40.08, 37.11
-8.28, 29.05
36.35, 7.28
12.5, -11.77
8.27, 10.89
49.33, 37.98
22.82, -11.16
23.5, 8.1
-41.62, 25.02
-47.02, 10.13
-1.9, -26.98
19.83, -0.27
11.45, 42.05
-24.42, -48.87
-19.9, 17.81
-29.74, -33.04
40.57, 16.0
-5.81, 39.17
-17.3, 16.59
-30.15, -6.91
38.03, -11.56
8.31, -18.35
-36.38, -0.35
33.71, 34.87
-22.32, -33.09
12.57, -0.61
-18.46, 33.91
48.2, -4.75
-42.53, -46.85
37.28, -45.85
20.86, 7.06
-19.1, 29.15
-48.09, -36.41
-4.52, -47.53
32.97, -26.26
12.92, -5.35
13.0, 15.5
30.74, 45.85
18.45, -30.07
-2.49, -32.13
-48.92, -2.78
21.42, -32.09
-22.76, -15.43
19.73, 2.04
11.44, 25.62
-10.65, 29.19
40.62, -41.28
43.26, 22.24
-37.01, -4.65
-12.32, 6.88
37.93, 29.68
44.43, -3.63
15.13, -29.51
22.19, 31.83
14.16, 21.77
-28.67, 40.0
3.7, 29.08
-17.96, 41.0
35.58, -15.15
-41.72, -5.91
5.03, 26.82
-1.26, -47.16
30.91, -43.59
-16.5, 28.79
-35.95, -35.13
1.65, 22.36
44.92, -41.4
-27.86, 2.67
-20.98, 22.88
13.89, 2.28
34.36, 6.0
-18.83, -11.88
34.53, 40.05
-29.18, 35.08
46.84, 2.42
7.3, -29.9
3.59, 0.32
10.52, -47.22
6.29, -0.9
19.1, -43.41
3.87, -8.62
45.69, 42.34
-23.08, -2.68
31.57, 40.06
-2.35, -18.28
-30.85, 11.79
42.53, -37.05
27.93, -47.72
18.7, -17.79
-14.47, 11.98
-39.51, 23.09
3.04, -6.32
-12.43, -8.66
2.93, -34.03
35.13, 11.17
35.68, -26.73
24.08, 31.05
40.27, -18.41
-28.19, 49.84
38.75, -36.61
-26.07, 22.66
-24.05, -40.3
33.22, -7.84
28.99, -37.4
-9.72, 18.52
18.24, 41.14
46.84, -38.46
0.57, 25.82
0.28, 18.57
-31.1, -42.94
-39.38, -46.26
-31.55, -29.61
34.02, 49.03
42.69, -40.48
-43.81, 45.15
-3.79, 26.47
-17.32, -3.3
20.1, 34.43
-31.87, -4.6
23.93, -9.47
1.26, -48.46
12.94, -9.55
9.96, 0.43
-24.17, 41.13
24.44, 27.8
31.46, -9.44
19.48, 26.73
22.26, -42.94
-48.94, -14.44
-26.79, 44.47
16.61, -16.22
15.98, 6.96
3.31, -11.04
49.99, 14.23
48.01, -47.72
-24.34, -9.84
-44.95, -30.46
-12.43, -40.16
46.71, 6.8
-45.49, 43.02
-34.01, -2.82
-33.09, -0.45
11.12, -44.15
44.53, -7.93
2.68, 9.78
-13.44, -21.42
-20.4, -48.6
-25.5, -45.72
-34.34, 25.46
-11.0, 39.75
24.84, -44.98
48.88, 44.45
-42.65, 40.56
-7.05, -2.24
47.32, -25.63
2.34, 43.73
22.27, -3.16
10.36, -38.49
-29.63, -44.79
2.81, -37.57
-5.71, 16.79
8.22, -8.05
27.8, 3.08
23.43, -26.16
-38.61, 39.27
28.43, 12.5
18.5, 6.49
9.17, 13.31
25.33, -31.02
-25.1, 47.97
-46.05, -43.92
-22.91, -7.48
12.33, -39.75
-41.36, 17.63
5.06, 13.11
-12.68, -2.14
-28.94, -15.63
24.48, 33.86
-42.57, -38.02
30.92, 12.37
26.88, -28.68
-7.56, -24.19
30.99, -13.11
15.37, 48.92
-17.46, 4.86
24.61, 42.08
-7.23, -13.07
-40.3, 37.52
-42.15, -41.7
18.53, -20.11
27.55, -42.39
-28.67, 16.21
-21.71, -35.71
-14.22, 22.6
-13.36, -38.26
41.86, 43.99
41.34, -6.2
30.3, -19.52
43.47, 39.47
-25.17, -13.83
-13.44, -13.67
-10.44, -11.24
-30.5, 6.38
29.71, 4.06
-32.34, 25.89
38.09, -21.85
-47.78, 1.56
4.42, 6.75
14.9, -25.45
-27.96, 26.54
-10.56, -16.22
-0.63, 3.73
41.5, -8.94
32.62, 16.67
35.35, 30.59
45.78, 14.03
30.23, -7.84
-7.95, -35.38
24.12, 49.1
-12.45, -33.23
-20.8, 46.98
-44.08, -19.16
27.59, -32.06
-43.77, -4.12
8.41, 40.93
-46.36, -39.13
9.48, -27.6
-31.5, -21.89
31.71, -2.05
-23.95, 38.74
41.43, -15.79
4.73, 45.7
-22.54, 49.02
-12.84, 29.68
21.33, 10.61
-17.82, -24.06
36.92, 1.61
-23.35, 3.46
-49.86, 32.14
-41.76, -23.01
21.65, -40.28
-1.99, -3.13
45.55, 8.66
35.73, -19.64
32.63, -29.16
4.34, 2.57
-34.24, 33.2
-18.86, -18.93
-3.28, 21.51
-39.42, -10.58
-3.82, 46.69
-48.77, -12.29
21.0, -26.25
-48.95, 49.16
29.95, -29.33
-12.4, 3.98
-20.18, -16.26
-10.78, 16.67
23.14, -17.06
44.5, 6.28
32.69, -40.77
-35.92, -40.56
17.77, 20.83
-32.01, -9.77
33.69, 9.27
-40.99, -27.34
-34.29, -37.59
-9.32, -42.73
26.69, 32.11
-8.79, -48.46
-48.15, -16.9
-15.78, 15.07
0.94, 28.86
32.52, 11.14
40.3, 4.83
-14.74, 0.04
-35.82, 21.34
48.68, 1.6
12.71, -30.21
7.65, 19.71
-17.08, 42.91
-37.6, 47.36
-36.44, 40.41
6.03, -23.56
40.93, 49.21
-37.73, 32.47
-25.91, 7.36
33.08, -31.44
4.79, -42.37
-46.8, -31.99
15.84, -19.24
17.12, 23.77
-11.84, 9.19
30.38, -48.37
-30.05, -3.2
-35.7, -11.36
6.95, -32.63
1.98, -23.63
6.8, -16.79
14.17, -46.21
17.1, -35.53
-3.01, -8.86
12.38, 18.93
25.8, 25.17
-1.42, 49.46
-9.1, -6.6
-17.93, -44.53
25.19, -19.52
9.34, -43.02
0.26, -10.33
-44.8, 19.49
2.62, -26.09
-26.41, -43.16
41.13, 46.65
16.57, 36.65
-37.62, 3.76
45.1, -49.94
-25.61, -20.07
8.55, -45.41
-19.22, -0.16
43.39, 47.73
-2.74, -29.35
-20.47, 42.26
39.68, -30.45
37.97, 49.54
-30.85, 38.02
21.31, -36.51
19.6, 43.78
-27.68, -19.28
16.46, 29.05
-12.75, 16.2
42.67, 16.7
-22.32, 13.99
-5.97, 2.83
-24.91, 30.32
25.57, -25.41
-11.53, 42.93
-0.01, -32.01
0.56, 34.54
45.15, -32.61
-5.96, 27.3
28.66, 29.11
-26.41, -1.09