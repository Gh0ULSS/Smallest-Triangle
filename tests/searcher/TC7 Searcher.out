read 426 points
46.84, 2.42
48.68, 1.60
45.91, -1.63
This is a triangle